### PR TITLE
Bug: Policy statement has trailing slash on bucket name

### PR DESCRIPTION
**Pull request by dvaldivia**
Fixes https://github.com/minio/console/issues/1218
Fixed https://github.com/minio/console/issues/1209

Signed-off-by: Daniel Valdivia <18384552+dvaldivia@users.noreply.github.com>

### CODE REVIEW

**Comment by harshavardhana** [comment]
does this fix #1209 as well?

**Comment by dvaldivia** [comment]
added a temporary fix for #1209

File: go.mod [hunk 1]
```diff
@@ -40,4 +40,115 @@ require (
...
+	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
+	k8s.io/klog/v2 v2.8.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7 // indirect
+	k8s.io/utils v0.0.0-20201110183641-67b214c5f920 // indirect
+	maze.io/x/duration v0.0.0-20160924141736-faac084b6075 // indirect
+	sigs.k8s.io/controller-runtime v0.8.0 // indirect
+	sigs.k8s.io/structured-merge-diff/v4 v4.1.0 // indirect
+	sigs.k8s.io/yaml v1.2.0 // indirect
+)
+
```
Comment (harshavardhana) [inline go.mod:153]: looks like a bug @dvaldivia - you are duplicating `require()`

**Review by harshavardhana — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (dvaldivia) [reply go.mod]: removed

**Review by Alevsk — APPROVED** [review]

**Review by Alevsk — APPROVED** [review]

**Review by bexsoft — APPROVED** [review]
LGTM

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**